### ISSUE TITLE
Capcom Arcade Stadium (1515950)

### ISSUE THREAD

**Issue by opal1974** [ValveSoftware/Proton#5086]
# Compatibility Report
- Name of the game with compatibility issues: Capcom Arcade Stadium
- Steam AppID of the game: 1515950

## System Information
- GPU:  GTX 1660
- Driver/LLVM version:  470.57.02
- Kernel version:  5.4.0-81
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: Proton Experimental

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 

Frame rate plummets when you get to the title screen, sometimes you can get into the game but then you start to get such slow speed it is not playable.  The test I made for this compat report I couldnt even get to the game, and got a crash before getting to the title screen.

## Reproduction

Install and play

[steam-1515950.log](https://github.com/ValveSoftware/Proton/files/7010939/steam-1515950.log)

**Comment by kisak-valve** [comment]
Hello @opal1974, is this game digging into swap until it runs out of system ram?

From the log: `grep "DXVK: v1.9" steam-1515950.log | wc -l` gives 1026. A game initializing DXVK over a thousand times seems abnormal.

**Comment by opal1974** [comment]
I am not sure, but everyone else seems to have the same issue (or a variation).  The game seemed to hard lock up for me when I was doing the test and I thought I would have to button reset the computer, but then it crashed.

**Comment by opal1974** [comment]
Noticed some people saying this works now, at least in 1943 I dont seem to have the slowdowns, but I am getting no sound now.

[steam-1515950.log](https://github.com/ValveSoftware/Proton/files/7749470/steam-1515950.log)

It behaves the same both in the experimental and current proton builds -

**Comment by opal1974** [comment]
There was an update a day or so ago, and now I can happily report that the game seems to work fine now, with sound.

**Comment by groenator** [comment]
I  cannot open the game on my steam deck. I always get the wine license and an error codes: 0x2000400 and 0x2000001.

**Comment by opal1974** [comment]
> I cannot open the game on my steam deck. I always get the wine license and an error codes: 0x2000400 and 0x2000001.

Are you sure you dont mean Capcom Arcade 2nd Stadium? I dont get that error on the first game (this entry) but I do with 2nd Stadium

**Comment by groenator** [comment]
You are correct, I get is in the second version. My bad! I just saw a few minutes ago another issue for the second version was opened 3 days ago.
